DelegateTest/Program.cs

```diff
@@ -24,6 +24,11 @@ static void Main()
            
         }
 
+        public static string GetDate()
```

Comment:
The date is current date, so method name can be GetCurrentDate()

Reply:
Method name has been changed